ui/admin/app/styles/app.scss

```diff
@@ -710,33 +710,61 @@
 }
 
 .mapping-table {
+  border: 0;
+
   /* stylelint-disable selector-class-pattern */
-  td > div,
-  .hds-form-field__control:not(:first-child) {
-    margin: 0;
+  .hds-table__thead {
+    .hds-table__th {
+      height: auto;
+      line-height: 1;
+    }
   }
 
-  .hds-table__tbody td {
-    padding-left: 0;
-  }
+  .hds-table__thead,
+  .hds-table__tbody {
+    .hds-table__tr {
+      // Remove background color on all cells (especially header cells)
+      background-color: transparent;
+    }
 
-  .hds-table__th {
-    padding: sizing.rems(xs) 0;
-    height: sizing.rems(l) + sizing.rems(xs);
-  }
+    .hds-table__th,
+    .hds-table__td {
+      padding: 0.75rem 1rem;
 
-  .hds-table__thead .hds-table__tr {
-    background-color: transparent;
-  }
-  /* stylelint-enable selector-class-pattern */
+      // Rely on right padding for cell spacing...
+      padding-left: 0;
 
-  label {
-    display: none;
-  }
+      &:last-child {
+        // ...except in the last cell, which doesn't need right padding
+        padding-right: 0;
 
-  border: 0;
+        // Shrink last cell to minimum possible width
+        width: 1%;
+        white-space: nowrap;
```

Comment:
What's the purpose of adding `nowrap` here, shouldn't we always be expecting a button of some sort?

Reply:
In case the button has multiple words in it.